doc/options.md

```diff
@@ -895,6 +896,47 @@ a
 }
 ```
 
+## rule-delimiter
+
+Number of line breaks after a ruleset.
+
+Acceptable values:
+
+* `{Number}` — number of newlines;
```

Comment:
I find it a bit confusing since other options which accept numbers treat them as spaces, not newlines